### PR TITLE
Re-enable counters

### PR DESCRIPTION
**Pull request by pitrou**
They were disabled by mistake, probably mine.

### CODE REVIEW

**Comment by mrocklin** [comment]
This is an odd exception to see in the error logs:

```
tornado.application - ERROR - Exception in callback functools.partial(<function wrap.<locals>.null_wrapper at 0x7fe9ae38eea0>, io_loop=<tornado.platform.epoll.EPollIOLoop object at 0x7fe9adbf1198>)
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/test-environment/lib/python3.5/site-packages/tornado/ioloop.py", line 605, in _run_callback
    ret = callback()
  File "/home/travis/miniconda/envs/test-environment/lib/python3.5/site-packages/tornado/stack_context.py", line 277, in null_wrapper
    return fn(*args, **kwargs)
TypeError: start() got an unexpected keyword argument 'io_loop'
tornado.application - ERROR - Exception in callback functools.partial(<function wrap.<locals>.null_wrapper at 0x7fe9adfcf400>, io_loop=<tornado.platform.epoll.EPollIOLoop object at 0x7fe9adbf1198>)
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/test-environment/lib/python3.5/site-packages/tornado/ioloop.py", line 605, in _run_callback
    ret = callback()
  File "/home/travis/miniconda/envs/test-environment/lib/python3.5/site-packages/tornado/stack_context.py", line 277, in null_wrapper
    return fn(*args, **kwargs)
TypeError: start() got an unexpected keyword argument 'io_loop'
tornado.application - ERROR - Exception in callback functools.partial(<function wrap.<locals>.null_wrapper at 0x7fe9ad999268>, io_loop=<tornado.platform.epoll.EPollIOLoop object at 0x7fe9adbf1198>)
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/test-environment/lib/python3.5/site-packages/tornado/ioloop.py", line 605, in _run_callback
    ret = callback()
  File "/home/travis/miniconda/envs/test-environment/lib/python3.5/site-packages/tornado/stack_context.py", line 277, in null_wrapper
    return fn(*args, **kwargs)
TypeError: start() got an unexpected keyword argument 'io_loop'
```

File: distributed/core.py [hunk 1]
```diff
@@ -98,23 +98,22 @@ def __init__(self, handlers, connection_limit=512, deserialize=True,
...
-            self.events = defaultdict(lambda: deque(maxlen=10000))
-            self.event_counts = defaultdict(lambda: 0)
-
-            pc = PeriodicCallback(self.monitor.update, 500, io_loop=self.loop)
-            self.loop.add_callback(pc.start)
-            if self.digests is not None:
-                self._last_tick = time()
-                self._tick_pc = PeriodicCallback(self._measure_tick, 20, io_loop=self.loop)
-                self.loop.add_callback(self._tick_pc.start)
+        # XXX?
```
Comment (mrocklin) [inline distributed/core.py]: Why the `XXX` here?

Comment (pitrou) [inline distributed/core.py]: It's a leftover from the previous version. I should remove it.

**Comment by pitrou** [comment]
> This is an odd exception to see in the error logs:

Uh. I may have mixed up my changes in this PR with something else. Let me see.

**Comment by mrocklin** [comment]
I'm taking a look at the `test_task_stream_clear_interval` failure

**Comment by mrocklin** [comment]
Any thoughts on the failure in the last travis-ci run @pitrou ?

**Comment by pitrou** [comment]
I think it's unrelated. I've opened an issue for it at https://github.com/dask/distributed/issues/1173 and I'm trying to find a potential fix (I can't reproduce here).

**Pull request merged**